tools/test-proxy/Azure.Sdk.Tools.TestProxy/Common/StringSanitizer.cs

```diff
@@ -1,14 +1,31 @@
 ﻿using System;
-using System.Collections.Generic;
-using System.Linq;
+using System.Net;
 using System.Text;
 using System.Text.RegularExpressions;
-using System.Threading.Tasks;
+
 
 namespace Azure.Sdk.Tools.TestProxy.Common
 {
     public static class StringSanitizer
     {
+        /// <summary>
+        /// Quick and easy abstraction for checking regex validity. Passing null explicitly will result in a True return.
+        /// </summary>
+        /// <param name="regex">A regular expression.</param>
+        /// <returns>True if the regex will successfully compile or if the regex is null. False otherwise.</returns>
```

Comment:
Void method so you can probably delete this line.

Reply:
Refactor. 👎

Reply:
```suggestion
```